webrtc.html

```diff
@@ -4946,6 +4947,23 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
               <code>degradationPreference</code> indicates which is
               preferred.</p>
             </dd>
+            <dt><dfn><code>upscaleAllowed</code></dfn> of type
+            <span class="idlMemberType"><a>boolean</a></span>,
+            defaulting to <code>false</code></dt>
+           <dd>
+             <p>When, and only when, this value is set true the browser MAY
+            upscale video temporally or spatially, or upscale the audio sampling
+            rate, or upscale the number of bits in the audio sample that this
+            <code><a>RtpSender</a></code> sends over RTP.
```

Comment:
I thought the only consensus we had in the WG was for resolution (spatial) and for video.  I don't recall any discussion about audio or temporal upscale.     

And even if we do have controls for those things, I  think they should be separate controls, not one big control for 3 different things.